build/pipelines/templates/helix-runtests-job.yml

```diff
@@ -0,0 +1,131 @@
+    inputs:
+      restoreSolution: build/Helix/packages.config
+      feedsToUse: config
+      nugetConfigPath: nuget.config
+      restoreDirectory: packages
+
+  - task: DownloadBuildArtifacts@0 
+    condition:
+      and(succeeded(),eq(variables['useBuildOutputFromBuildId'],''))
+    inputs: 
+      artifactName: ${{ parameters.artifactName }} 
+      downloadPath: '$(artifactsDir)'
+
+  - task: DownloadBuildArtifacts@0 
+    condition:
+      and(succeeded(),ne(variables['useBuildOutputFromBuildId'],''))
+    inputs: 
+      buildType: specific
+      buildVersionToDownload: specific
+      project: $(System.TeamProjectId)
+      pipeline: ${{ parameters.useBuildOutputFromPipeline }}
+      buildId: $(useBuildOutputFromBuildId)
+      artifactName: ${{ parameters.artifactName }} 
+      downloadPath: '$(artifactsDir)'
+
+  - task: CmdLine@1
+    displayName: 'Display Artifact Directory payload contents'
+    inputs:
+      filename: 'dir'
+      arguments: '/s $(artifactsDir)'
+
+  - task: powershell@2
+    displayName: 'PrepareHelixPayload.ps1'
+    inputs:
+      targetType: filePath
+      filePath: build\Helix\PrepareHelixPayload.ps1
+      arguments: -Platform '$(buildPlatform)' -Configuration '$(buildConfiguration)' -ArtifactName '${{ parameters.artifactName }}'
+      
+  - task: CmdLine@1
+    displayName: 'Display Helix payload contents'
+    inputs:
+      filename: 'dir'
+      arguments: '/s $(Build.SourcesDirectory)\HelixPayload'
+
+  - template: helix-createprojfile-steps.yml
+    parameters:
+      condition: and(succeeded(),ne('${{ parameters.testSuite }}','NugetTestSuite'))
+      testFilePath: '$(artifactsDir)\${{ parameters.artifactName }}\$(buildConfiguration)\$(buildPlatform)\Test\TerminalApp.LocalTests.dll'
+      outputProjFileName: 'RunTestsInHelix-TerminalAppLocalTests.proj'
+      testSuite: '${{ parameters.testSuite }}'
+      taefQuery: ${{ parameters.taefQuery }}
+      
+  - template: helix-createprojfile-steps.yml
+    parameters:
+      condition: and(succeeded(),ne('${{ parameters.testSuite }}','NugetTestSuite'))
+      testFilePath: '$(artifactsDir)\${{ parameters.artifactName }}\$(buildConfiguration)\$(buildPlatform)\Test\Conhost.UIA.Tests.dll'
+      outputProjFileName: 'RunTestsInHelix-HostTestsUIA.proj'
+      testSuite: '${{ parameters.testSuite }}'
+      taefQuery: ${{ parameters.taefQuery }}
+
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish generated .proj files'
+    inputs:
+      PathtoPublish: $(Build.ArtifactStagingDirectory)
+      artifactName: ${{ parameters.artifactName }}
+      
+  - task: DotNetCoreCLI@2
+    displayName: 'Run tests in Helix (open queues)'
+    env:
+      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+    inputs:
+      command: custom
+      projects: build\Helix\RunTestsInHelix.proj
+      custom: msbuild
+      arguments: '$(helixCommonArgs) /p:IsExternal=true /p:Creator=WinUI /p:HelixTargetQueues=$(openHelixTargetQueues)'
```

Comment:
is this `Creator` correct?

Reply:
No. I'll try flipping it to Terminal and hope it's just metadata. But if it actually blocks Helix from submitting with that name, I'll have to flip it back.

Reply:
(at least until Keith and I work it out with the Helix team, I'll file a follow on if I can't make it say `Terminal`.)